### ISSUE TITLE
[Docs v4.1] Input group - missing ID for aria-describedby

### ISSUE THREAD

**Issue by AlessandroVecchi** [twbs/bootstrap#26548]
https://getbootstrap.com/docs/4.1/components/input-group/#button-addons

**aria-describedby="basic-addon1"**
**aria-describedby="basic-addon2"**
```
<div class="input-group mb-3">
  <div class="input-group-prepend">
    <button class="btn btn-outline-secondary" type="button">Button</button>
  </div>
  <input type="text" class="form-control" placeholder="" aria-label="" aria-describedby="basic-addon1"> 
</div>

<div class="input-group mb-3">
  <input type="text" class="form-control" placeholder="Recipient's username" aria-label="Recipient's username" aria-describedby="basic-addon2">
  <div class="input-group-append">
    <button class="btn btn-outline-secondary" type="button">Button</button>
  </div>
</div>

<div class="input-group mb-3">
  <div class="input-group-prepend">
    <button class="btn btn-outline-secondary" type="button">Button</button>
    <button class="btn btn-outline-secondary" type="button">Button</button>
  </div>
  <input type="text" class="form-control" placeholder="" aria-label="" aria-describedby="basic-addon1"> <---
</div>

<div class="input-group">
  <input type="text" class="form-control" placeholder="Recipient's username" aria-label="Recipient's username" aria-describedby="basic-addon2">
  <div class="input-group-append">
    <button class="btn btn-outline-secondary" type="button">Button</button>
    <button class="btn btn-outline-secondary" type="button">Button</button>
  </div>
</div>
```

**Comment by patrickhlauke** [comment]
Good spot. that whole section seems a bit messed-up as a result of multiple changes, edits, copy/pastes over the years. Will be fixed by https://github.com/twbs/bootstrap/pull/26551
